gammapy/data/observations.py

```diff
@@ -336,7 +336,7 @@ def obs_info(self):
     @property
     def pointing(self):
         if self._pointing is None:
-            self._pointing = FixedPointingInfo.from_fits_header(self.obs_info)
+            self._pointing = FixedPointingInfo.from_fits_header(self.events.table.meta)
```

Comment:
The pointing information should come logically from the data stored in the HDU pointing, no?

Reply:
If there is one yes, but so far we have only fixed pointing which is stored in the events header.

Reply:
How is pointing information relevant for the WCD experiments?

Reply:
I think this is fine, for HAWC/SWGO we would just set the FixedPointingInfo to fixed_altaz (so in DRIFT mode). Would also be nice to not need a pointing HDU if the pointing is fixed anyway.